runtime/common/src/lib.rs

```diff
@@ -146,6 +152,30 @@ pub mod types {
 	)]
 	#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 	pub struct InstanceId(pub u128);
+
+	bitflags::bitflags! {
+		/// The current admin roles we support
+		#[derive(codec::Encode, codec::Decode,  TypeInfo)]
+		pub struct AdminRoles: u32 {
+			const POOL_ADMIN = 0b00000001;
+			const BORROWER  = 0b00000010;
+			const PRICING_ADMIN = 0b00000100;
+			const LIQUIDITY_ADMIN = 0b00001000;
+			const MEMBER_LIST_ADMIN = 0b00010000;
+			const RISK_ADMIN = 0b00100000;
+		}
+	}
+
+	#[derive(codec::Encode, codec::Decode, TypeInfo, Debug, Clone, Eq, PartialEq)]
+	pub struct TrancheInvestors(pub(crate) u32);
+
+	/// The structure that we store in the pallet-permissions storage
+	/// This here implements trait Properties.
+	#[derive(codec::Encode, codec::Decode, TypeInfo, Clone, Eq, PartialEq, Debug)]
+	pub struct PermissionRoles {
```

Comment:
I would like if we move these to either common libs or bring the PoolRole here. Seems two tightly connected with each other

Reply:
I have thought about this too. I don't want this in the common-libs, as it clearly is a runtime-type. And I did not want to move the `PoolRole` here, as we then have dependency in our pallets on `runtime-common`. 😄 

But I also think you got a valid point here. I am fine with leaving it as it is, but if we want to change something I would move `PoolRole` here.

Reply:
Probably my main reason against having it in the common-libs is that it does not belong into the traits module. But maybe its time to open a types module. Then we can move it there. WDYT?

Reply:
either is fine by me as long the crates dont inter depend on each other